holidays/countries/__init__.py

```
@@ -210,4 +210,4 @@
 from holidays.countries.vietnam import Vietnam, VN, VNM
 from holidays.countries.yemen import Yemen, YE, YEM
 from holidays.countries.zambia import Zambia, ZM, ZMB
-from holidays.countries.zimbabwe import Zimbabwe, ZW, ZWE
+from holidays.countries.zimbabwe import Zimbabwe, ZW, ZWE
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)